src/components/MerchCard/MerchCard.jsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react"
+import { Form, Row, Col, Button, Alert } from "react-bootstrap" 
+import { metricToCustomary } from "../../utils/conversion"
+import "./MerchCard.css"
+
+export default function MerchCard(props) {
+    // unit enum 
+    const UNIT_TYPE = Object.freeze({
+        VOLUME: 1,
+        WEIGHT: 0,
+        UNKNOWN: -1
+    })
+    // merch enum 
+    const MERCH_TYPE = Object.freeze({
+        ONSALE: 1,
+        INSTOCK: 0,
+        WISHLIST: -1
+    })
+    const quantityWithUnits = props.isMetric ? props.item.quantity : metricToCustomary(props.item.quantity, props.item.unitType)
+    var unit
+    if (props.item.unitType == UNIT_TYPE.VOLUME) {
+        unit = props.isMetric ? "milliliters" : "cups"
+    } else if (props.item.unitType == UNIT_TYPE.WEIGHT) {
+        unit = props.isMetric ? "grams" : "ounces"
+    } else {
+        unit = "count"
+    }
```

Comment:
This logic seems like it belongs somewhere else. Is there a central place for dealing with units?

Reply:
Yup I agree. I put the function into MarketPlace.jsx so I can pass it as props to multiple components